app/models/change.rb

```diff
@@ -1,7 +1,7 @@
 class Change < Sequel::Model
   plugin :time_machine
 
-  def self.cleanup(older_than: Date.current.ago(3.months))
+  def self.cleanup(older_than: 3.months.ago.beginning_of_day)
```

Comment:
for consistency this needs `.to_date` on the end to convert to a `Date` object? Not sure if it would make any difference but this does change from `Date` to `TimeWithZone`